docs/src/basics/client-server.md

```diff
@@ -1,25 +1,48 @@
-# Client / Server
+# Architecture
 
-Zeebe uses a client / server architecture. There are two main components: *Broker* and *Client*.
+There are four main components in Zeebe's architecture: the client, the gateway, the broker, and the exporter.  
 
-![client-server](/basics/client-server.png)
+![zeebe-architecture](/basics/zeebe-architecture.png)
+
+## Client
+
+Clients are libraries that you embed in an application (e.g. a microservice that executes your business logic) to connect to a Zeebe cluster. Clients have two primary uses:
+* Carrying out business logic (starting workflow instances, publishing messages, working on tasks)
+* Handling operational issues (updating workflow instance payloads, resolving incidents)
+
+More about Zeebe clients:
+* Clients connect to the Zeebe gateway via [gRPC](https://grpc.io), which uses http/2-based transport. To learn more about gRPC in Zeebe, check out the [gRPC section of the docs](/grpc/README.html).
+* The Zeebe project includes officially-supported Java and Go clients, and gRPC makes it possible to generate clients in a range of different programming languages. [Community clients](https://awesome.zeebe.io) have been created in other languages, including C#, Ruby, and JavaScript.
+* Client applications can be scaled up and down completely separately from Zeebe--the Zeebe brokers do not execute any business logic.
+
+## Gateway
+
+The gateway, which proxies requests to brokers, serves as a single entry point to a Zeebe cluster. 
+
+The gateway is stateless and sessionless, and gateways can be added as necessary for load balancing and high availability.
 
 ## Broker
 
-A Zeebe broker (server) has three main responsibilities:
+The Zeebe broker is the distributed workflow engine that keeps state of active workflow instances.
+
+Brokers can be partitioned for horizontal scalability and replicated for fault tolerance. A Zeebe deployment will often consist of more than one broker.
```

Comment:
```suggestion
Brokers can be partitioned for horizontal scalability and replicated for fault tolerance. A Zeebe cluster will often consist of more than one broker.
```

I know what you mean with Deployment  but it might confuse with the deployments in Zeebe :sweat_smile: